ui/src/submissions/conferences/components/__tests__/ExistingConferencesAlert.test.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ExistingConferencesAlert from '../ExistingConferencesAlert';
+
+describe('ExistingConferencesAlert', () => {
```

Comment:
we need to test `onDatesChange `. However,  for now there is no straight forward way to test `useEffect`, when you have a component that has containers in the subtree. Because you need a `mount`, which requires you to wrap the component with `Provider`, then you can use `wrapper.setProps` because it would reference to `Provider`.

TLDR; put a  comment ```// TODO: test onDatesChange when issue of testing useEffect with shallow rendering is resolved```, I will do little investigation and we'll add those after.